Backend/Controllers/AuthenticationController.cs

```diff
@@ -113,8 +113,8 @@ public IQueryable<Session> GetSessions([FromAuthentication] User user)
 
     [RequiresAuthentication]
     [HttpPost("change-password-uwu")]
-    public async Task PasswordChange([FromAuthentication] User user, string newPassword) {
-        ChangePassword(user, newPassword);
+    public async Task PasswordChange([FromAuthentication] User user, ChangePasswordDto newPassword) {
+        ChangePassword(user, newPassword.NewPassword);
```

Comment:
1. Should not send the duplicate password for no reason
2. Requiring the correct old password should harden security, so we should do it